docs/demo/matplotlib/intro.md

```diff
@@ -46,6 +46,8 @@ given as explicit values. Can you adjust each setting to match the goal plot
 below? The two plots are compared on the bottom right, with differences
 highlighted in red.
 
+The blue line is 2.5 points wide and the striped orange line is 1 point wide.
```

Comment:
I intended the student to find these values by experimenting with the code, seeing the interplay between what they write and what matplotlib draws. Did you find that too frustrating?

Reply:
Somewhat, the difference is too small to notice it in the graph (e.g. 2.4 and 2.6 versus 2.5). Because it's the first exercise, I would consider it as a hint to get started.